model/task/results.go

```diff
@@ -133,3 +134,38 @@ func FilterTasksOnStatus(tasks []Task, statuses ...string) []Task {
 
 	return out
 }
+
+type Result struct {
+	Name  string `json:"name"`
+	Count int    `json:"count"`
+}
+type ResultCountList struct {
+	Total              []Result `json:"total"`
+	Inactive           []Result `json:"inactive"`
+	Unstarted          []Result `json:"unstarted"`
+	Started            []Result `json:"started"`
+	Succeeded          []Result `json:"succeeded"`
+	Failed             []Result `json:"failed"`
+	SetupFailed        []Result `json:"setup-failed"`
+	SystemFailed       []Result `json:"system-failed"`
+	SystemUnresponsive []Result `json:"system-unresponsive"`
+	SystemTimedOut     []Result `json:"system-timed-out"`
+	TestTimedOut       []Result `json:"test-timed-out"`
+}
+
+func GetResultCountList(results []StatsList) ResultCountList {
+	list := ResultCountList{}
+	listVal := reflect.ValueOf(&list).Elem()
```

Comment:
Is there a way to do this without reflection?

Reply:
no longer uses reflection